### ISSUE TITLE
Experimental Fn-Lock feature not working on GA401IV

### ISSUE THREAD

**Issue by moracabanas** [seerge/g-helper#504]
**Bug Description**
Experimental Fn-Lock feature where F1 - F12 will trigger corresponding Fn+Fx hotkeys without Fn does not work on GA401IV apparently. 

**Clear scenario to Reproduce**
1. Go to `Laptop Keyboard` > `Extra`
2. Click on `Process Fn+F hotkeys without FN` checkbox

**Expected behavior**
You should be able to make use of Function keys (F) without FN

**App Logs**
01-Jun-23 5:26:01 PM: ------------
01-Jun-23 5:26:01 PM: App launched: ROG Zephyrus G14 GA401IV_GA401IV :0.74.0.0en-US
01-Jun-23 5:26:01 PM: NVIDIA GeForce RTX 2060 with Max-Q Design
01-Jun-23 5:26:01 PM: AutoSetting for Online
01-Jun-23 5:26:01 PM: BatteryLimit = 100 : OK
01-Jun-23 5:26:01 PM: BatteryLimit = 100 : OK
01-Jun-23 5:26:01 PM: PerformanceMode = 1 : OK
01-Jun-23 5:26:01 PM: GET GPU CLOCKS: -250, 0
01-Jun-23 5:26:01 PM: FanCPU = 14-3F-44-48-4C-50-54-62-11-1A-22-29-34-43-51-5A : OK
01-Jun-23 5:26:01 PM: FanGPU = 14-3F-44-48-4C-50-54-62-16-1F-26-2D-39-47-55-5F : OK
01-Jun-23 5:26:01 PM: Boost 4
01-Jun-23 5:26:01 PM: Eco flag : -65536
01-Jun-23 5:26:01 PM: Mux flag : -65536
01-Jun-23 5:26:01 PM: Screen = 120Hz : OK
01-Jun-23 5:26:01 PM: ScreenOverdrive = 1 : 0
01-Jun-23 5:26:01 PM: Matrix Off
01-Jun-23 5:26:06 PM: Monitor Power On
01-Jun-23 5:26:06 PM: AutoSetting for Online
01-Jun-23 5:26:06 PM: BatteryLimit = 100 : OK
01-Jun-23 5:26:06 PM: PerformanceMode = 1 : OK
01-Jun-23 5:26:06 PM: GET GPU CLOCKS: -250, 0
01-Jun-23 5:26:06 PM: FanCPU = 14-3F-44-48-4C-50-54-62-11-1A-22-29-34-43-51-5A : OK
01-Jun-23 5:26:06 PM: FanGPU = 14-3F-44-48-4C-50-54-62-16-1F-26-2D-39-47-55-5F : OK
01-Jun-23 5:26:06 PM: Boost 4
01-Jun-23 5:26:06 PM: Eco flag : -65536
01-Jun-23 5:26:06 PM: Mux flag : -65536
01-Jun-23 5:26:06 PM: Screen = 120Hz : OK
01-Jun-23 5:26:06 PM: ScreenOverdrive = 1 : 0
01-Jun-23 5:26:06 PM: Matrix Off
01-Jun-23 5:26:43 PM: USB-KB 64|32HID-compliant vendor-defined device\\?\hid#vid_0b05&pid_1866&mi_02&col03#8&3397f86&0&0002#{4d1e55b2-f16f-11cf-88cb-001111000030}:5D-B3-00-01-FF-FF-FF-EB-00-00-FF-FF-FF-00-00-00-00

**Insights about logs**
I could identify the last line is generated when I press ⏩ (F4) trying this functionality:
```
01-Jun-23 5:26:43 PM: USB-KB 64|32HID-compliant vendor-defined device\\?\hid#vid_0b05&pid_1866&mi_02&col03#8&3397f86&0&0002#{4d1e55b2-f16f-11cf-88cb-001111000030}:5D-B3-00-01-FF-FF-FF-EB-00-00-FF-FF-FF-00-00-00-00
```

**Desktop (please complete the following information):**
 - OS: Windows 11 
 - Laptop model GA401IV

**Asus software**
- Armoury crate -> Installed but *disabled*
- MyASUS Not Installed
-  All ASUS services disabled except `ASUS Optimization` & `ASUS On-Screen Display`

**Comment by seerge** [comment]
@moracabanas disable asus optimization service OR download [0.75 ](https://github.com/seerge/g-helper/releases/tag/v0.75)

**Comment by moracabanas** [comment]
Could I disable asus optimization sevice If I want the `enabled/disabled` mic overlay and functionality?
I really want to end with 0% ASUS bloatware in the long term on my laptop 😄

*Bro, you should be PROUD of your work.* This well built asus laptops are just messed with the crappy software
Thanks you a lot

**Comment by seerge** [comment]
@moracabanas optimization service is already added to "debloat.bat", just re-run it again. I can't (at least easily) disable it from the app, cause that requires admin permissions.

I just realized you have 2020 model, and it has different actions for F2-F4 (media keys), while rest has backlight control there.

Try this build, it should act properly even for F2-F4
[GHelper.zip](https://github.com/seerge/g-helper/files/11627172/GHelper.zip)

**Comment by seerge** [comment]
@moracabanas and yes, if optimization service is disable app will take over all M1-M4 hotkeys, including mic mute

**Comment by moracabanas** [comment]
It works! Thanks you for your fast repy

**0.75.0.0 logs when ⏩ and 🎤 pressed**
```logs
01-Jun-23 5:46:14 PM: ------------
01-Jun-23 5:46:14 PM: App launched: ROG Zephyrus G14 GA401IV_GA401IV :0.75.0.0en-US
01-Jun-23 5:46:14 PM: NVIDIA GeForce RTX 2060 with Max-Q Design
01-Jun-23 5:46:14 PM: AutoSetting for Online
01-Jun-23 5:46:14 PM: BatteryLimit = 100 : OK
01-Jun-23 5:46:14 PM: BatteryLimit = 100 : OK
01-Jun-23 5:46:14 PM: PerformanceMode = 1 : OK
01-Jun-23 5:46:14 PM: GET GPU CLOCKS: 0, 0
01-Jun-23 5:46:15 PM: FanCPU = 14-3F-44-48-4C-50-54-62-11-1A-22-29-34-43-51-5A : OK
01-Jun-23 5:46:15 PM: FanGPU = 14-3F-44-48-4C-50-54-62-16-1F-26-2D-39-47-55-5F : OK
01-Jun-23 5:46:15 PM: Eco flag : -65536
01-Jun-23 5:46:15 PM: Mux flag : -65536
01-Jun-23 5:46:15 PM: Screen = 120Hz : OK
01-Jun-23 5:46:15 PM: ScreenOverdrive = 1 : 0
01-Jun-23 5:46:15 PM: Matrix Off
01-Jun-23 5:46:15 PM: KB Backlight:5D-BA-C5-C4-FF
01-Jun-23 5:46:15 PM: Monitor Power On
01-Jun-23 5:46:23 PM: FnLock = 0 : 0
01-Jun-23 5:46:28 PM: F4
01-Jun-23 5:46:52 PM: WMI event 124
```

ASUS optimization is to not interfering but, when I disabled it, OSD is not displayed so I will keep it until GHelper comes with a native OSD for it so I can see my mic state (enabled/disabled)

**Comment by seerge** [comment]
@moracabanas but g-helper shows OSD when mic is disabled/enabled :)

**Comment by moracabanas** [comment]
Yes, It seems I didn't restart after getting rid of `ASUS optimization service` I was custom-loading that crappy service at startup with a `Startup Task`.
Then for caution, I rerun `debloat.bat`.

PD:  I also sight-read some of your code, and I found this valuable [link](https://www.asus.com/support/FAQ/1047338/) which fixes the lack-of-driver issue I was facing for a week when I upgrade to Windows 11. 
I ended up getting some random `AsusControlInterfaceV3` from somewhare because I wasn't capable find an official solution and It worked. 
So I notice you actually check for that driver, and I used the link to upgrade it. 
Thanks for that 😄👍
